docs/src/componentDocs/Grade/playground/PlaygroundPage.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { Grade, GradeProps } from '@brightlayer-ui/react-native-components';
+import Stack from '@mui/material/Stack';
+import Box from '@mui/material/Box';
+import {
+    InputConfig,
+    PreviewComponent,
+    CodeSnippetFunction,
+    getPropsToString,
+    getPropsMapping,
+    Playground,
+} from '@brightlayer-ui/react-doc-components';
+import 'prismjs/components/prism-jsx.min';
+
+const inputConfig: InputConfig = [
+    // Required Props
+    {
+        id: 'label',
+        type: 'string',
+        typeLabel: `string`,
+        initialValue: 'A+',
+        description: 'The text shown in the circle',
+        required: true,
+        category: 'Required Props',
+    },
+    // Optional Props
+    {
+        id: 'fontColor',
+        type: 'color',
+        initialValue: 'white',
```

Comment:
we need to verify if initialValue should have hex color

Reply:
it should not

Reply:
Added defaultValue.